srtp/srtp.c

```diff
@@ -769,13 +769,13 @@ static inline int base_key_length(const srtp_cipher_type_t *cipher, int key_leng
   case SRTP_AES_ICM_256:
     /* The legacy modes are derived from
      * the configured key length on the policy */
-    return key_length - 14;
+    return key_length - SRTP_SALT_LEN;
```

Comment:
Several statements that follow return a computed value, but they should be constants.  Why do a computation here?  Is this to help spot errors? Anything checking for errors?